lib/src/ast_extensions.dart

```diff
@@ -0,0 +1,228 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
+
+extension AstNodeExtensions on AstNode {
+  /// The comma token immediately following this if there is one, or `null`.
+  Token? get commaAfter {
+    var next = endToken.next!;
+    if (next.type == TokenType.COMMA) return next;
+
+    // TODO(sdk#38990): endToken doesn't include the "?" on a nullable
```

Comment:
This issue is closed

https://github.com/dart-lang/sdk/issues/38990

Can this TODO be addressed?